userdocs/src/getting-started/install.md

```diff
@@ -1,26 +1,19 @@
-To download the latest release, make sure you have the required pre-requisites. Then run the command matching your
-operating system.
 
-## Prerequisites
 
-## Linux
+## Install on Linux
 
 ```bash
 curl --silent --location "https://github.com/oslokommune/okctl/releases/latest/download/okctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
 sudo mv /tmp/okctl /usr/local/bin
 ```
 
-## macOS
-
-### Prerequisites
-
-### Installation
+## Install on macOS
 
 ```bash
 brew tap oslokommune/tap
 brew install oslokommune/tap/okctl
 ```
 
-## Shell autocompletion
+## Enable Shell autocompletion
```

Comment:
Shell -> shell ? Or is it a name?

Reply:
here i'm refering to the gas station

Reply:
Ah then it makes sense. I love autocompletion when I'm at Shell.